Dockerfile

```diff
@@ -144,4 +144,13 @@ RUN sudo apt-get -y install gcovr && \
 # Add engineering tools scripts to image
 ADD --chown=carma ./code_coverage /home/carma/.ci-image/engineering_tools/code_coverage
 
-ENTRYPOINT [ "/home/carma/.base-image/entrypoint.sh" ]
+RUN sudo apt-get install -y sqlite3 autotools-dev automake
+
+RUN sudo git clone https://github.com/OSGeo/PROJ.git && \
```

Comment:
We definitely don't want to just be running a raw `git clone` here, the version that gets pulled down could change over time without our awareness. You either want to check out how the other repo's `checkout.sh` scripts work and do the same to check out one and only one specific commit, or try to download and extract a specific release from a zip as we do for armadillo above. Also make sure it's done in the `/home/carma`  home folder, i think this compiles everything directly under `/` as it stands which is more clutter than we need.